client/context.go

```diff
@@ -370,10 +370,10 @@ func (ctx Context) PrintOutput(toPrint interface{}) error {
 		out, err = yaml.Marshal(&toPrint)
 
 	case "json":
+		out, err = ctx.JSONMarshaler.MarshalJSON(toPrint)
```

Comment:
Since this is now using `JSONMarshaler` we don't actually need to deprecate this in favor of `Println` do we? Is that right @alexanderbez ?

Reply:
Good question. I can't recall why the TODO was added. I think we'll still need this, no?

Reply:
The main difference is that `Println` can use `ctx.Output` if it's set instead of stdout, which I think makes sense anyway. I'm doing some fixes to `Println` in #6367 and I'm thinking to just update `PrintOutput` instead and remove `Println`.